attributes/default.rb

```diff
@@ -1,5 +1,5 @@
 default[:apiaxle][:environment]                            = 'development'
-default[:apiaxle][:version]                                = '1.12.34'
+default[:apiaxle][:version]                                = '1.12.35'
```

Comment:
That's just a separate bump to the latest release of apiaxle. Not much changed, and I figured that it would be better to track the latest release version. I probably should have put it in a separate pr, but this was easier and somewhat related anyway.